doc/source/ray-core/monitoring-debugging/tools.rst

```diff
@@ -0,0 +1,171 @@
+Troubleshooting Tools and Features
```

Comment:
This will be merged with the current top level `Observability`?

Reply:
Yes. I think this will explain a high level concept, and I am planning to have each section in `Observability` to go in detail

Reply:
will be done in a follow up